Project-Bootcamp/querys/query.js

```diff
@@ -0,0 +1,66 @@
+var $ = require ('jquery');
+var _ = require ('lodash');
+
+var Apikey = 'qrmacznjya3vdnynqdr4trf3bd9t3fau';
+var Locale = 'en_US';
+
+var BlizzardAPI = function () {
+
+    this.apikey = Apikey;
+    this.locale = Locale;
+};
+
+BlizzardAPI.prototype.RealmStatus = function (callback) {
+
+    $.ajax({
+        url: 'https://us.api.battle.net/wow/realm/status?locale=' + this.locale + '&apikey=' + this.apikey,
+        type: 'get',
+        datatype: 'json',
+        success: function (response) {
+            callback (response.realms);
+        }
+    })
+};
+
+BlizzardAPI.prototype.CharacterInfo = function (realm, name, callback) {
+
+    $.ajax({
+        url: 'https://us.api.battle.net/wow/character/' + this.realm + '/' + this.name
```

Comment:
**Minor:** I like that you broke lines, but just a minor suggestion: what we use to follow as a coding standard is to leave  the operator in the same line, e.g. something like:
```
url: 'https://us.api.battle.net/wow/character/' + this.realm + '/' + this.name +
    '?fields=stats%2C+items%2C+pvp%2C+guild' + '&locale=' + this.locale +
    '&apikey=' + this.apikey,
```